### PR TITLE
EBMEDS-1279: Add DSV service to docker-compose

### PR DESCRIPTION
**Pull request by joonatanv**
Adds DSV to the docker-compose definitions as a part of integrating it to the rest of Sirppi. [EBMEDS-1279](https://jira.duodecim.fi/browse/EBMEDS-1279)

---

api-gateway changes: https://github.com/ebmeds/api-gateway/pull/91

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -146,6 +146,17 @@ services:
       mode: replicated
       replicas: 1
 
+  dsv:
+    # listens internally on port 3010 by default
+    image: "quay.io/duodecim/ebmeds-diagnosis-specific-view:${EBMEDS_VERSION}"
```
Comment (SkylarPhx) [inline docker-compose.yml:151]: Should the image name be `ebmeds-dso`?

File: docker-compose.yml [hunk 2]
```diff
@@ -146,6 +146,17 @@ services:
       mode: replicated
       replicas: 1
 
+  dsv:
```
Comment (SkylarPhx) [inline docker-compose.yml:149]: Hmm..`dso` or `dsv`?

**Review by SkylarPhx — COMMENTED** [review]
Looks good just a minor question about naming the image.

File: docker-compose.yml (hunk 2 above)
Comment (joonatanv) [reply docker-compose.yml:149]: Yeah... Perhaps it should be `dso` already. I've already made a ticket for the rename. Engine is using `dsp` currently, the repository was named `diagnosis-specific-view`, and the actual name would be `dso`.

File: docker-compose.yml (hunk 1 above)
Comment (joonatanv) [reply docker-compose.yml:151]: That'd be cleaner, sure.

File: docker-compose.yml (hunk 2 above)
Comment (joonatanv) [reply docker-compose.yml:149]: https://jira.duodecim.fi/browse/EBMEDS-1309 Unify naming scheme for DSO ticket created.

File: docker-compose.yml (hunk 1 above)
Comment (joonatanv) [reply docker-compose.yml:151]: https://jira.duodecim.fi/browse/EBMEDS-1309 Unify naming scheme for DSO ticket created.

**Review by SkylarPhx — APPROVED** [review]

**Review by jouko-kelomaki-solita — APPROVED** [review]

**Review by Tomkinen — APPROVED** [review]

**Pull request merged**